scripts/setup/ubuntu-20.04/install_deps.sh

```diff
@@ -10,6 +10,7 @@ sudo apt-get --yes update \
   bison \
   cmake \
   curl \
+  ctags \
```

Comment:
Are we keeping it this in alphabetical order

Reply:
Done.